gravitee-am-ui/src/app/domain/settings/theme/theme.component.ts

```diff
@@ -0,0 +1,127 @@
+    { 'name': 'deep-purple', 'primaryButtonColorHex': '#6A4FF7', 'primaryTextColorHex': '#FFFFFF', 'secondaryButtonColorHex': '#DAD3FD' },
+    { 'name': 'purple', 'primaryButtonColorHex': '#9346E9', 'primaryTextColorHex': '#FFFFFF', 'secondaryButtonColorHex': '#E4D1FA' },
+    { 'name': 'light-purple', 'primaryButtonColorHex': '#CC63D6', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#F2D8F5' },
+    { 'name': 'deep-pink', 'primaryButtonColorHex': '#D8549A', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#F5D4E6' },
+    { 'name': 'pink', 'primaryButtonColorHex': '#FE8AA8', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#FFE2E9' },
+    { 'name': 'deep-red', 'primaryButtonColorHex': '#C52852', 'primaryTextColorHex': '#FFFFFF', 'secondaryButtonColorHex': '#F1C9D4' },
+    { 'name': 'deep-orange', 'primaryButtonColorHex': '#D94C3E', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#F6D2CF' },
+    { 'name': 'orange', 'primaryButtonColorHex': '#EA9345', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#FAE4D0' },
+    { 'name': 'yellow', 'primaryButtonColorHex': '#EED052', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#FBF3D4' },
+    { 'name': 'light-green', 'primaryButtonColorHex': '#9FC929', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#E7F2C9' },
+    { 'name': 'green', 'primaryButtonColorHex': '#7FCF79', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#DFF3DE' },
+    { 'name': 'teal', 'primaryButtonColorHex': '#66C2AC', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#D9F0EA' },
+    { 'name': 'grey', 'primaryButtonColorHex': '#8A979E', 'primaryTextColorHex': '#000000', 'secondaryButtonColorHex': '#E2E5E7' },
+    { 'name': 'deep-grey', 'primaryButtonColorHex': '#252829', 'primaryTextColorHex': '#FFFFFF', 'secondaryButtonColorHex': '#C8C9C9' }
+  ];
+  selectedColorPalette: string;
+  config: any = { lineNumbers: true };
+  formChanged: boolean;
+
+  constructor(private snackbarService: SnackbarService,
+              private router: Router,
+              private route: ActivatedRoute,
+              private authService: AuthService,
+              private themeService: ThemeService,
+              private dialogService: DialogService) {
+  }
+
+  ngOnInit() {
+    this.envId = this.route.snapshot.params['envHrid'];
+    this.domain = this.route.snapshot.data['domain'];
+    this.themes =  this.route.snapshot.data['themes'] || [];
+    this.theme = this.themes[0] || {};
+    if (this.theme.id && this.theme.primaryButtonColorHex) {
+      const filteredObj = _.find(this.colorPalettes, { 'primaryButtonColorHex' : this.theme.primaryButtonColorHex });
+      if (filteredObj) {
+        this.selectedColorPalette = filteredObj.name;
+      }
+    }
+    if (this.theme.logoWidth != undefined && this.theme.logoWidth === 0) {
```

Comment:
Shouldn't it be better to use '!==' instead of '!='? Possibly we don't expect any type conversion here.

Reply:
FP